src/com/android/volley/Request.java

```diff
@@ -699,4 +704,17 @@ public void setJoined(boolean joined) {
     public void expireCache() {
         // no - op
     }
+
+    public boolean isExecuting() {
+        return mRequestQueue.isProcessing(this);
+    }
+
+    public void setFinished(boolean finished) {
```

Comment:
I can't have finished be true for the response delivery if I don't have a method to set it. Since finish is called after the delivery